client/components/Page.jsx

```diff
@@ -12,29 +12,74 @@ const createPad = (day) => {
 //Page Element Styles
 
 const Month = styled.div`
+#reservation-calendar & {
```

Comment:
Whats the point of repeating the style for both div tags? Is it just because your styling applies to both the widget and the calendar?

Reply:
yes they need to exist in both portals